js/ui/popup.js

```diff
@@ -413,6 +416,18 @@ var Popup = Overlay.inherit({
             .toggleClass(POPUP_NORMAL_CLASS, !value);
     },
 
+    _shouldUseContentAutoResizableClass() {
+        if(this.option("height") !== "auto" || this.option("shading") || this.option("fullScreen") || this.option("showTitle") || this._getToolbarItems("bottom").length) {
```

Comment:
Why the popup should not be auto-resized if shading/showTitle/bottom toolbar is enabled? I think it is good when the dxPopup will be resized in all cases when the user sets the `auto` height